src/client/api/types.ts

```diff
@@ -111,6 +113,15 @@ export type PythonApi = {
      * Registers a visibility filter for the interpreter status bar.
      */
     registerInterpreterStatusFilter(filter: IInterpreterStatusbarVisibilityFilter): void;
+    getCondaVersion?(): Promise<SemVer | undefined>;
+    /**
+     * Returns the conda executable.
+     */
+    getCondaFile?(): Promise<string | undefined>;
+    getEnvironmentActivationShellCommands?(
```

Comment:
New API exposed by python extension. Possible users don't have the latest python extenison hence we need to take that into account.